src/tvheadend/IHTSPDemuxPacketHandler.h

```diff
@@ -8,7 +8,7 @@
 
 #pragma once
 
-#include "kodi/DemuxPacket.h"
+#include "kodi/addon-instance/inputstream/DemuxPacket.h"
```

Comment:
@AlwinEsch Travis could not find this file: https://travis-ci.org/github/kodi-pvr/pvr.hts/jobs/740043332#L754

Reply:
The last build and merge was a bit fast before related Kodi version was available on Launchpad.

But related Ubuntu build is available, see https://launchpad.net/~team-xbmc/+archive/ubuntu/xbmc-nightly/+sourcepub/11731082/+listing-archive-extra.

Reply:
It failed with DEBIAN_BUILD=true on bionic and focal. Could you retrigger the build?